### ISSUE TITLE
[docs] https://docs.lilypad.tech/ redirects to https://docs.lilypad.tech/lilypad

### ISSUE THREAD

**Issue by AquiGorka** [Lilypad-Tech/lilypad-docs#37]
This is not the best for SEO. Definitely a minor concern but ideally we keep the current version docs at `/`

**Comment by developersteve** [comment]
Ahh after checking the DNS it looks like this is because the docs are structured within a `lilypad` folder within the repo and being carried over from gitbooks.

**Comment by developersteve** [comment]
See https://github.com/Lilypad-Tech/lilypad-docs/tree/main

**Comment by developersteve** [comment]
Adding in @PBillingsby here too for the gitbook side of things

**Comment by developersteve** [comment]
@PBillingsby I think this might be able to fix it if we changed the `.gitbook.yml` root folder location to `./lilypad` as per https://docs.gitbook.com/integrations/git-sync/content-configuration#root

**Comment by AquiGorka** [comment]
Thank you for looking into this. Definitely low priority.

**Comment by PBillingsby** [comment]
@developersteve Can we set up a redirect once the appended `/lilypad` is removed? So redirect from `/lilypad` to `/`.

We have a lot of references to https://docs.lilypad.tech/lilypad and I believe once we change the root path all of those links will be broken.

**Comment by developersteve** [comment]
> @developersteve Can we set up a redirect once the appended `/lilypad` is removed? So redirect from `/lilypad` to `/`.
> 
> We have a lot of references to https://docs.lilypad.tech/lilypad and I believe once we change the root path all of those links will be broken.

Yup that was the plan, the `.gitbook.yml` was added per the spec doc but it didn't change so is still in the same state. From memory this was going to need a complete refactor to work :(
